### PR TITLE
added Draggable class with sortable functionality

### PR DESCRIPTION
**Pull request by marsh73**

### CODE REVIEW

File: scarlet/cms/static/scarlet/source/js/views/Draggable.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Draggable {
+	constructor(el) {
+		this.$el = $(el)
+		this.init()
+	}
+
+	init () {
+		let draggable = this.$el.find("tr[draggable]").parents("tbody")
```
Comment (annaocd) [inline scarlet/cms/static/scarlet/source/js/views/Draggable.js]: use single quotes

File: scarlet/cms/static/scarlet/source/js/views/Draggable.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+		draggable.sortable({
+			stop: function (e, ui) {
+				let inputs = $(ui.item.context.offsetParent).find(":text")
+
+				for (var i = 0; i < inputs.length; i++) {
```
Comment (annaocd) [inline scarlet/cms/static/scarlet/source/js/views/Draggable.js]: consider storing inputs.length in a var

File: scarlet/cms/static/scarlet/source/js/views/Select.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import Backbone, { View } from 'backbone'
```
Comment (annaocd) [inline scarlet/cms/static/scarlet/source/js/views/Select.js]: is Backbone import needed, or just View?

**Comment by annaocd** [comment]
👍

File: scarlet/cms/static/scarlet/source/js/views/Select.js (hunk 3 above)
Comment (marsh73) [inline scarlet/cms/static/scarlet/source/js/views/Select.js]: just view... been trying to clean those up as i notice them lingering

**Pull request merged**